.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: php
 
 php:
-    - 5.4
-    - 5.5
-    - 5.6
     - 7.0
+    - 7.1
+    - 7.2
     - hhvm
```

Comment:
If @tedivm is ok with it, given both  Stash and Symfony 4 has dropped support for this, you can swap this for `7.3` instead.

Reply:
👍

Reply:
but SF 3.4 shoud still work on 7.1 and 7.2?

Reply:
I thought he was referring to the "hhvm" tests, which I think we can safely drop at this point.

Reply:
yes, to be explicit 
```suggestion
    - 7.3
```

Reply:
ah haha because you tagged the lines with 7.1 and 7.2 :grin: it is not really the scope of this PR but I will change it :+1: